### PR TITLE
Sanity Unit Tests

### PR DESCRIPTION
**Pull request by stephenrlouie**
### Change log

- Just to get more test coverage, yes a lot of that is fairly simple code. Just wanted simple sanity tests.
- Better exclusions on the Makefile for testing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 EXECUTABLE ?= sigma
 IMAGE ?= bin/$(EXECUTABLE)
-PACKAGES = $(shell go list ./... | grep -E -v '/bin|/vendor|/cmd|/docs|/api/v1/gen')
+PACKAGES = $(shell go list ./... | grep -E -v '/bin|/vendor|/cmd|/docs|/api/v1/gen|/api/v1/client|/database/mock_database|/task/mock_task|/plugin/examples')
```
Comment (adamschaub) [inline Makefile]: What's the reason for this?

File: model/model_test.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+func TestGet(t *testing.T) {
+	m := MetaImpl{}
+
+	// Get a nil key
+	a, ok := m.Get("a")
+	if ok {
+		t.Error("Did not return an error when expected")
+	}
+	if a != nil {
+		t.Error("Shold be nil")
```
Comment (adamschaub) [inline model/model_test.go]: Should

File: model/model_test.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+func TestGetString(t *testing.T) {
+	m := MetaImpl{}
+
+	// GetString on a nil key
+	a, ok := m.GetString("a")
+	if ok {
+		t.Error("Did not return an error when expected")
+	}
+	if a != "" {
+		t.Error("Shold be nil")
```
Comment (adamschaub) [inline model/model_test.go]: NIT: Should

File: model/model_test.go [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+	if !ok {
+		t.Error("Could not get key")
+	} else {
+		if a != 1 {
+			t.Error("Did not properly set key")
+		}
+	}
+}
+
+func TestGetString(t *testing.T) {
```
Comment (adamschaub) [inline model/model_test.go]: What happens when you `GetString` on a non-string value that has been previously added?

File: Makefile (hunk 1 above)
Comment (stephenrlouie) [inline Makefile]: to prevent those files from showing up when you run `make test`

File: model/model_test.go (hunk 4 above)
Comment (stephenrlouie) [inline model/model_test.go]: tested now, good call!

File: Makefile (hunk 1 above)
Comment (adamschaub) [inline Makefile]: deal

**Comment by adamschaub** [comment]
G2G

**Pull request merged**